Extension/src/LanguageServer/extension.ts

```diff
@@ -498,7 +501,7 @@ function installVsix(vsixLocation: string): Thenable<void> {
             }
         }(platformInfo);
         if (!vsCodeScriptPath) {
-            return Promise.reject(new Error('Failed to find VS Code script'));
+            return Promise.reject(new Error(localize('failed.to.find.scipt', 'Failed to find VS Code script')));
```

Comment:
Most of the strings related to processing the Insiders update are not user-facing.  They are for telemetry and should not be localized.